### ISSUE TITLE
Try all `./t/*.rakutest` files

### ISSUE THREAD

**Issue by pheix** [pheix/raku-trove#1]
From https://www.reddit.com/r/rakulang/comments/zlnugw/day_14_trove_yet_another_tap_harness_pheix_team/ by @melezhik

> Hi! Trove looks really interesting. The one "issue" I have with it is a user has to insert `raku t/test.rakutest` for every test under `t/` directory, that's tedious on my view. Would Trove allow something like `raku t/*.rakutest` as a test step within config file?
>
> Other than that this is a great tool and I am looking forward to play with Trove and SparrowCI integration...

**Comment by pheix** [comment]
I added the feature to explore some folder and build stages from matched members.

The config is:

```
target: Trove dynamic tests
explore:
  base: ./t
  pattern: (<[013]>+).(t|rakutest)
  interpreter: raku
  recursive: 1
```

It's passed both on [github](https://github.com/pheix/raku-trove/actions/runs/3800716944/jobs/6464450612#step:9:1) and [gitlab](https://gitlab.com/pheix/raku-trove/-/jobs/3534323414#L70).

By default `interpreter` is set to `raku` and `recursive` is switched off, so these members could be ommited, `pattern` is just a Raku regexpr.

**Comment by melezhik** [comment]
I tested it with sparrow - https://ci.sparrowhub.io/report/2375 , my test case seems does see a configuration ?

WARNING: no stages in configuration file are found

**Comment by melezhik** [comment]
please hold on, should have tested from `explore-feature` branch

**Comment by melezhik** [comment]
trove test works OK in Sparrow - https://ci.sparrowhub.io/report/2383

**Comment by pheix** [comment]
Included in [Explore feature release v0.0.33](https://github.com/pheix/raku-trove/releases/tag/v0.0.33), commit: 4542d75b219434a06492bcac36239a0aa8972711

Details: https://github.com/pheix/raku-trove#configuration-file-sections

**Comment by melezhik** [comment]
Cool! Thanks. Will continue to play with trove )))
